config/debug/manager_debug_patch.yaml

```diff
@@ -1,14 +1,17 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: kusk-gateway-manager
   namespace: system
 spec:
   template:
     spec:
       containers:
         - name: manager
           resources: null
+          # Disable healtchecks for the debuging image
+          livenessProbe: null
+          readinessProbe: null
```

Comment:
Is this to allow the pod running the debug image to become ready immediately?

Reply:
Yes